### PR TITLE
Bump kubevirtci to start testing k8s-1.19 provider

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
This commit bump the kubevirtci to the latest version so the
k8s-1.19 cluster can be used to test kubevirt

It also do a cherry-pick of https://github.com/kubevirt/kubevirt/pull/4176 to fix selinux issues.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Bump kubevirtci to k8s-1.19
```

### CODE REVIEW

**Comment by qinqon** [comment]
I see the following at the operator pod at `test_id:3148` that fails 

```{"component":"virt-operator","level":"info","msg":"http: TLS handshake error from 192.168.66.101:40228: local error: tls: bad record MAC\n","pos":"server.go:3085","timestamp":"2020-09-09T09:29:42.764595Z"}
```

**Comment by qinqon** [comment]
Looks like at `BeforeSuite` cert rotation parameters are changed at `AdjustKubeVirtResource`

This are the values:

```golang
kv.Spec.CertificateRotationStrategy = v1.KubeVirtCertificateRotateStrategy{SelfSigned: &v1.KubeVirtSelfSignConfiguration{
        CARotateInterval:   &metav1.Duration{Duration: 10 * time.Minute},       
        CertRotateInterval: &metav1.Duration{Duration: 7 * time.Minute},        
        CAOverlapInterval:  &metav1.Duration{Duration: 4 * time.Minute},        
    }}                
```

Doing so re-construct the `Service` that back the delete validation webhook, this means that clusterIP is changed, maybe the CA ips are not updated at k8s-1.19 somehow, so HTTPS is rejected.

**Comment by qinqon** [comment]
k8s-1.19 logs 

https://prow.apps.ovirt.org/log?id=1304012758051721216&job=pull-kubevirt-e2e-k8s-1.19

**Comment by qinqon** [comment]
Testing with new k8s-1.19 container digest that upgrade calico, https://github.com/kubevirt/kubevirtci/pull/442, looks like it's fixing the failing test.

**Comment by qinqon** [comment]
Looks like there were a bug at kubernetes itself they are fixing it right now, https://github.com/kubevirt/kubevirtci/pull/443.

The prow run with this https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4149/pull-kubevirt-e2e-k8s-1.19/1305384168758185984

**Comment by jean-edouard** [comment]
@qinqon could you please cherry-pick https://github.com/kubevirt/kubevirt/pull/4176?
This PR and the one above depend on each other.
Thanks!

**Comment by qinqon** [comment]
@jean-edouard done, thanks!

**Review by jean-edouard — COMMENTED** [review]
/approve

**Comment by dhiller** [comment]
Something is wrong here?

```
The cluster provider k8s-1.19 does not exist
```
source: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4149/pull-kubevirt-e2e-k8s-1.19/1305534159782940672#1:build-log.txt%3A45

**Comment by qinqon** [comment]
/test pull-kubevirt-e2e-k8s-1.19

Forgot to add the `cluster-up/cluster/k8s-1.19` directory :-/

**Comment by dhiller** [comment]
@qinqon still something wrong with the bump, when it tries to auto sync kubevirtci it stumbles over [cluster-up-sha.txt](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4149/pull-kubevirt-generate/1305743666450534400)

**Comment by qinqon** [comment]
@dhiller calling again `make bump-kubevirtci` let's see now.

**Comment by qinqon** [comment]
k8s-1.19: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4149/pull-kubevirt-e2e-k8s-1.19/1306573255875760128

File: Makefile
Comment (dhiller) [inline Makefile:150]: This is still here? Wasn't this supposed to be removed?

**Review by dhiller — COMMENTED** [review]
Suspicious that the test test_id:3090 is failing for two lanes though, but it seems to be a flake: kubevirt/kubevirt#4009

I don't really understand why we need to run the bump dockerized?

File: Makefile
Comment (qinqon) [reply Makefile:150]: We discover that when you don't runt it dockerized the cluster-up digest is different between ci and users, that was the issue with generate job.

**Review by dhiller — APPROVED** [review]
/lgtm

**Comment by dhiller** [comment]
/approve
/hold

Let's wait for another reviewer.

**Comment by qinqon** [comment]
Pushed chagens that remove the need to dockerize `make bump-kubevirtci` looks like we have use the same locales at sort to make it canonical

**Comment by qinqon** [comment]
/hold cancel

We are not changing `make bump-kubevirtci` so we good.

**Comment by qinqon** [comment]
/test pull-kubevirt-e2e-k8s-1.17 

Not related.

**Pull request merged**
